server/controllers/finance/journal/index.js

```diff
@@ -514,3 +514,26 @@ function count(req, res, next) {
     })
     .catch(next);
 }
+
+
+/**
+ * PUT /journal/comments
+ * @param {object} params - { uuids: [...], comment: '' }
+ */
+function commentPostingJournal(req, res, next) {
+  const params = req.body.params;
+  const uuids = params.uuids.map((uuid) => {
+    return db.bid(uuid);
+  });
+
+  const sql = 'UPDATE posting_journal SET comment = ? WHERE uuid IN ?';
+  db.exec(sql, [params.comment, [uuids]])
+    .then((rows) => {
+      if (!rows.affectedRows || rows.affectedRows !== uuids.length) {
+        throw new BadRequest('Error on update general ledger comment');
```

Comment:
This should say "journal" not "general ledger".

It also needs to have a second parameter which is a key to be translated on the client